pkg/controller/pxc/replication.go

```diff
@@ -86,6 +92,193 @@ func (r *ReconcilePerconaXtraDBCluster) removeOutdatedServices(cr *api.PerconaXt
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) reconcileReplication(cr *api.PerconaXtraDBCluster) error {
+	if cr.Status.PXC.Ready < 1 || len(cr.Spec.PXC.ReplicationChannels) == 0 {
+		return nil
+	}
+
+	sfs := statefulset.NewNode(cr)
+
+	list := corev1.PodList{}
+	if err := r.client.List(context.TODO(),
+		&list,
+		&client.ListOptions{
+			Namespace:     cr.Namespace,
+			LabelSelector: labels.SelectorFromSet(sfs.Labels()),
+		},
+	); err != nil {
+		return errors.Wrap(err, "get pod list")
+	}
+
+	primary, err := r.getPrimaryPod(cr)
+	if err != nil {
+		return errors.Wrap(err, "get primary pxc pod")
+	}
+
+	for _, pod := range list.Items {
+		if pod.Status.PodIP == primary || pod.Name == primary {
+			primary = fmt.Sprintf("%s.%s.%s", pod.Name, sfs.Name(), cr.Namespace)
+			break
+		}
+	}
+
+	sort.Slice(list.Items, func(i, j int) bool {
+		return list.Items[i].Name > list.Items[j].Name
+	})
+
+	var primaryPod corev1.Pod
+	for _, pod := range list.Items {
+		if strings.HasPrefix(primary, fmt.Sprintf("%s.%s.%s", pod.Name, sfs.Name(), cr.Namespace)) {
+			primaryPod = pod
+			break
+		}
+	}
+
+	user := "root"
```

Comment:
operator user should be used
```suggestion
	user := "operator"
```

Reply:
07768ad6

Reply:
07768ad6